public/app/partials/signup_step2.html

```diff
@@ -6,7 +6,10 @@ <h3 class="p-b-1">You're almost there.</h3>
     <div class="p-b-1">
       We just need a couple of more bits of<br> information to finish creating your account.
     </div>
-    <form name="signUpForm" class="login-form gf-form-group">
+
+    <signup-page model="formModel" verifyEmailEnabled="verifyEmailEnabled" autoAssignOrg="autoAssignOrg"></signup-page>
+
+    <form name="signUpForm" class="login-form gf-form-group"> 
```

Comment:
Can we remove this page & partial?  Not sure we need to use angular wrapper for sign-up page, we can just make SignUp page a react container? Or maybe that is the plan and this is just a way to make sure the two forms work the same

Reply:
Yeah, this is just while doing the migration